GeosCore/fullchem_mod.F90

```diff
@@ -1583,6 +1494,15 @@ SUBROUTINE Do_FullChem( Input_Opt,  State_Chm, State_Diag,                 &
        CALL DEBUG_MSG( '### CHEMDR: after UCX_H2SO4PHOT' )
     ENDIF
 
+    ! Set State_Chm arrays for surface J-values used in HEMCO and
```

Comment:
These updates (1) pull out many calls into chemistry_mod and (2) save JNO2 and JOH for subsequent use.